.env

```diff
@@ -5,25 +5,25 @@ REACT_APP_TRAININGS_URL=
 REACT_APP_DNS=
 REACT_APP_GIT_PATH=
 REACT_APP_PASSWORD=
-REACT_APP_GRAFANA_BASE_URI=
+REACT_APP_MONITORING_BASE_URI=
```

Comment:
This is one of the configuration that could probably be moved to the region configuration.  
I'm think it's ok to keep it there for now but it will surely move to region configuration (defined by API) soon